pkg/controller/job/jobcontroller.go

```diff
@@ -540,7 +540,7 @@ func newCondition(conditionType batch.JobConditionType, reason, message string)
 	}
 }
 
-// getStatus returns no of succeeded and failed pods running a job
+// getStatus returns number of succeeded and failed pods in a running job
```

Comment:
This doesn't appear to be a typo. This also seems to read like you pass a job to this function, when you're actually passing in pods.